src/components/sponsor/SponsorList.jsx

```diff
@@ -1,15 +1,42 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import ReactHtml from 'raw-html-react'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Card from 'react-bootstrap/Card'
+import fallBackImage from '../../images/fallBackImage.svg'
+import Container from 'react-bootstrap/esm/Container'
 
 const SponsorList = ({ sponsors }) => (
-  <div className='sponsors'>
-    {sponsors.map(sponsor => (
-      <div key={sponsor.id} className='sponsor'>
-        <div className='title'>{sponsor.title}</div>
-        {JSON.stringify(sponsor)}
-      </div>)
-    )}
-  </div>
+  <Container className='my-md-5'>
+    <Row className='sponsors'>
+      <Col xs={12} className='my-5'>
+        {/sponsor/.test(window.location.href) ? <h1>Sponsors</h1> : <h2>Sponsors</h2>}
```

Comment:
This component should not do this kind of checks. Stuff like this should be passed with props or maybe even better by creating a `SponsorListPromoted` component similar to `SpeakerListPromoted`.